### DISCUSSION TITLE
UserId context with GitLab feature flags

### DISCUSSION THREAD

**Discussion by yuri-apanasik** [Unleash/unleash#6688] [Q&A]
We are using feature flags in GitLab, Unleash proxy (Docker) and Unleash proxy client (Angular version).
We with to try 'WithUserId' strategy, but looks like it doesn't work in our case. Looks like Unleash proxy ignores context set from client.
Am I right? Is it limitation of integration with GitLab?

**Comment by sighphyre** [comment]
Hey @yuri-apanasik,

Unfortunately, we don't control Gitlab's implementation of feature flags, so debugging that is very difficult for us and we also don't have an officially supported Angular SDK. I can try to help out but I'll need a bit more context.

The proxy should not be ignoring set context fields, but it might be a bug in the way the Angular SDK is setting those fields. Would you mind sharing which implementation of the community Angular client you're using? There's at least two that I'm aware of.

Are you using an up to date version of the Unleash Proxy?

If this is a limitation in Gitlab, you might have to raise a support ticket with them
